services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/ProdDiagnosisKeyBundlerKeyRetrievalTest.java

```diff
@@ -46,13 +47,16 @@
 
 @EnableConfigurationProperties(value = DistributionServiceConfig.class)
 @ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = {DistributionServiceConfig.class, ProdDiagnosisKeyBundler.class},
+@ContextConfiguration(classes = {DistributionServiceConfig.class, KeySharingPoliciesChecker.class, ProdDiagnosisKeyBundler.class},
     initializers = ConfigFileApplicationContextInitializer.class)
 class ProdDiagnosisKeyBundlerKeyRetrievalTest {
 
   @Autowired
   DistributionServiceConfig distributionServiceConfig;
 
+  @Autowired
+  KeySharingPoliciesChecker sharingPoliciesChecker;
```

Comment:
unused

Reply:
Yeah, there was a problem with the application context loading before that autowiring was added. I have to check if it is needed or not.